pre-commit/github_email_verify.sh

```diff
@@ -1,41 +1,49 @@
 #!/usr/bin/env bash
 
-LOCAL_NAME=$(git config --get user.name)
-LOCAL_EMAIL=$(git config --get user.email)
+LOCAL_NAME=$(git config --local user.name)
+LOCAL_EMAIL=$(git config --local user.email)
 
 GLOBAL_NAME=$(git config --global user.name)
 GLOBAL_EMAIL=$(git config --global user.email)
 
 GITHUB=$(git config --get remote.origin.url | grep -c github.com)
 
-if [ "$GITHUB" -eq 1 ] && [[ "$LOCAL_EMAIL" == "$GLOBAL_EMAIL" ]]; then
+if [ "$GITHUB" -eq 1 ] && [[ -z "$LOCAL_EMAIL" ]]; then
   # assigns stdin
   exec < /dev/tty
 
   printf \
-"[WARN] Trying to commit to a github repo using a private email
-Local git config
-\tuser.name:\t%s
-\tuser.email:\t%s
+"[WARN] Trying to commit to a github repo using global settings
 Global git config
 \tuser.name:\t%s
 \tuser.email:\t%s\n\n" \
-"$LOCAL_NAME" "$LOCAL_EMAIL" "$GLOBAL_NAME" "$GLOBAL_EMAIL"
+"$GLOBAL_NAME" "$GLOBAL_EMAIL"
 
-  read -p "[WARN] Proceed with private email [$LOCAL_EMAIL] (y/n): " -r OPTION
+  read -p "[WARN] Use global email [$GLOBAL_EMAIL] for this repo (y/N): " -r OPTION
 
   if [[ $OPTION == [yY] ]]; then
-    exit 0
+    git config --local user.email "$GLOBAL_EMAIL"
+    printf "\n[OK] Please run the commit again\n"
+    exit 1
   else
-    read -p "Enter new local user.email: " -r NEW_EMAIL
-
-    read -p "Proceed with new local user.email [$NEW_EMAIL] (y/n): " -r OPTION
+    
+    if [[ ! -z "$PRE_COMMIT_GITHUB_EMAIL" ]]; then
+      read -p "Enter new local user.email (leave empty to use '$PRE_COMMIT_GITHUB_EMAIL'): " -r NEW_EMAIL
```

Comment:
Sounds a bit better like this:

`read -p "Enter new local user.email or leave blank to proceed with [$PRE_COMMIT_GITHUB_EMAIL]: " -r NEW_EMAIL`